packages/itmat-job-executor/test/testFiles/JSONCurator.json

```diff
@@ -0,0 +1,83 @@
+{
+  "sub1":{
```

Comment:
As @sou-chon poined out, this might not be the most appropriate structure for this file. While this exactly reflects the database collection schema, it is impractical for users to make and will dramatically increase the bandwidth usage over the network.